metis/api/admin/v1/admin.proto

```
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package admin.v1;
```

Buf (1.72.0)

[error] Files with package "admin.v1" must be within a directory "admin/v1" relative to root but were in directory "metis/api/admin/v1".

(PACKAGE_DIRECTORY_MATCH)